Core/Main/elastixlib.cxx

```diff
@@ -46,9 +46,9 @@ namespace elastix
  * ******************* Constructor ***********************
  */
 
```

Comment:
I hate to say it but GitHub puts a hard line break in commit subjects of more 72 chars: "ENH: Initialize SmartPointers with nullptr instead of 0 which is ambi… \n …guous". Better keep the first line of a commit message at <= 72 chars, if possible.

Reply:
Sorry that I did not fix `elastixlib.cxx` before (as part of the upgrade to ITK5). I only tested ELASTIX_BUILD_EXECUTABLE = ON.